src/DS/sds.c

```diff
@@ -988,8 +997,16 @@ static int ds_sds_compose_add_component_source_with_ref(xmlDocPtr doc, xmlNodePt
 	xmlNodePtr cref_parent;
 
 	bool extended_component = false;
+    bool source_is_remote;
 
-	oscap_document_type_t doc_type = oscap_source_get_scap_type(component_source);
+    if (oscap_str_startswith(filepath, "com.")) {
```

Comment:
This is suspicious for me. Is it remote when the file path starts with "com."? 

Also, ds_sds_compose_add_component_source_with_ref is called from ds_sds_compose_add_component_with_ref which is called from ds_sds_compose_add_component which is used when composing datastreams.
What if I have a local file `com.example.xml` in current working directory  and I run this:
`oscap ds sds-compose com.example.xml ds.xml`

Reply:
>  Is it remote when the file path starts with "com."?

No. I just used a hardcoded string, and a dumb match to it just to see how the output would lookl ike. This PR is very rough, take it with a grain of salt, :)